### PR TITLE
Support passing multiple Fastly services

### PR DESCRIPTION
**Pull request by titosand**
This PR allows users to pass multiple Fastly service ID's (via a comma-separated string) to the exporter. 

It also adds an additional `service` label to all metrics, where the name of the service is derived from the [Fastly API.](https://docs.fastly.com/api/config.html#service_c967d3038e6c7cfb74bed38fa338a77b)

### CODE REVIEW

**Comment by msuterski** [comment]
Is the default rate limit on the Fastly API tokens going to allow for pulling stats from multiple services using a single token? How many would it be?

**Comment by titosand** [comment]
Per the API docs, `Read requests, such as GET and HEAD requests, are not limited.`

**Comment by neufeldtech** [comment]
Hi all,
I'm really interested in having this land so that I can try it out with multiple services - is there anything blocking this PR from being merged?

**Comment by titosand** [comment]
No blockers besides waiting for someone with write access to merge. I would recommend checking out to the PR branch if you want to try it out for now.

**Comment by swalberg** [comment]
We've been running this for a month now and it's awesome.

**Comment by nodox** [comment]
What is the status of this PR?

**Comment by titosand** [comment]
It just needs to be merged by someone who can, cc @peterbourgon

**Comment by peterbourgon** [comment]
Oh sorry, I wasn't watching this repo for some reason :) Will review now.

File: main.go
Comment (peterbourgon) [inline main.go:76]: This would become e.g. 

```go
for _, service := range *services {
    servicecopy := service
    g.Add(...
```

File: fastly.go
Comment (peterbourgon) [inline fastly.go:48]: We shouldn't lookup service name in the hot loop. I reckon this should be done once, at program start, and passed to queryLoop.

**Review by peterbourgon — CHANGES_REQUESTED** [review]

File: fastly.go
Comment (neufeldtech) [reply fastly.go:48]: I second this note. The service lookup in the hot loop is likely abusing the Fastly API, especially when running this with many (100+) services. (They've reached out to us about this issue in particular)

**Comment by peterbourgon** [comment]
I'll take a stab at implementing this myself today.

**Comment by titosand** [comment]
Thanks for the review and helping clean this up, I haven't had the time to get started.

**Comment by peterbourgon** [comment]
Closed in favor of #4, but thank you for the idea and contribution!

**Comment by peterbourgon** [comment]
https://github.com/peterbourgon/fastly-exporter/releases/tag/v0.0.3

**Pull request closed without merging**
